extras/bindings/c/include/precice/SolverInterfaceC.h

```diff
@@ -287,16 +282,7 @@ PRECICE_API void precicec_setMeshTetrahedra(
 /**
  * @brief Returns true (!=0), if data with given name is available.
  */
-PRECICE_API int precicec_hasData(const char *dataName, int meshID);
-
-/**
- * @brief Returns the data id belonging to the given name.
- *
- * The given name (dataName) has to be one of the names specified in the
- * configuration file. The data id obtained can be used to read and write
- * data to and from the coupling mesh.
- */
-PRECICE_API int precicec_getDataID(const char *dataName, int meshID);
+PRECICE_API int precicec_hasData(const char *meshName, const char *dataName);
```

Comment:
The order of input arguments seem to be changed here, now its mesh first and data second. Is this intentional?

Reply:
Yes, the order used to be consistent with the order of `getDataID`. Now it's consistent with the arguments of read/write data.